libs/brig-types/test/unit/Test/Brig/Types/TURN.hs

```diff
@@ -2,16 +2,44 @@
 
 module Test.Brig.Types.TURN where
 
-import Brig.Types.TURN hiding (turnURI)
+import Brig.Types.TURN
 import Data.Aeson
+import Data.List (sort)
+import Data.Misc
+import System.Random.Shuffle
 import Test.Brig.Types.Arbitrary ()
 import Test.Tasty
+import Test.Tasty.HUnit
 import Test.Tasty.QuickCheck
 
 tests :: TestTree
 tests = testGroup "TURN"
     [ testProperty "TurnURI: decode . encode = id" turnURIid
+    , testGroup "Order"
+        [ testCase "Turn order example" turnURIOrder
+        ]
     ]
 
 turnURIid :: TurnURI -> Property
 turnURIid t = Just t === (decode . encode) t
+
+turnURIOrder :: IO ()
+turnURIOrder = go (100 :: Int)
```

Comment:
Perhaps you could add a test case ensuring that shuffling a list of turnURIs with different hosts keeps the random order of the hosts, even if sorting the schemes and transports?

Reply:
Ah yes, good idea 👍